### PR TITLE
Return job steps from slurmdb_jobs.get #167

### PR DESCRIPTION
**Pull request by vkantchev**

### CODE REVIEW

File: pyslurm/pyslurm.pyx [hunk 1]
```diff
@@ -5467,10 +5468,100 @@ cdef class slurmdb_jobs:
                 JOBS_info[u'start'] = job.start
                 JOBS_info[u'state'] = job.state
                 JOBS_info[u'state_str'] = slurm.slurm_job_state_string(job.state)
+                
+                # TRES are reported as strings in the format `TRESID=value` where TRESID is one of:
+                # TRES_CPU=1, TRES_MEM=2, TRES_ENERGY=3, TRES_NODE=4, TRES_BILLING=5, TRES_FS_DISK=6, TRES_VMEM=7, TRES_PAGES=8
+                # Example: '1=0,2=745472,3=0,6=1949,7=7966720,8=0'
+                JOBS_info[u'stats'] = {}
+                stats = JOBS_info[u'stats']
+                stats[u'act_cpufreq']               = job.stats.act_cpufreq
```
Comment (giovtorres) [inline pyslurm/pyslurm.pyx]: Could we keep the standard Python spacing, i.e. one space surrounding the `=`?

File: pyslurm/slurm.pxd [hunk 2]
```diff
@@ -2781,7 +2781,34 @@ cdef extern from 'slurm/slurmdb.h' nogil:
         List tres_list
         List user_list
 
-    # ctypedef struct slurmdb_step_rec_t
+    ctypedef struct slurmdb_step_rec_t:
```
Comment (giovtorres) [inline pyslurm/slurm.pxd]: :clap:

File: pyslurm/pyslurm.pyx [hunk 3]
```diff
@@ -5467,30 +5467,30 @@ cdef class slurmdb_jobs:
                 JOBS_info[u'show_full'] = job.show_full
                 JOBS_info[u'start'] = job.start
                 JOBS_info[u'state'] = job.state
-                JOBS_info[u'state_str'] = slurm.slurm_job_state_string(job.state)
+                JOBS_info[u'state_str'] = slurm.stringOrNone(slurm.slurm_job_state_string(job.state), '')
```
Comment (giovtorres) [inline pyslurm/pyslurm.pyx]: Does `slurm_job_state_string` ever return `null`?

Comment (vkantchev) [reply pyslurm/pyslurm.pyx]: It returns byte array, e.g. b'123' which breaks things like json.dumps.

**Pull request merged**